Engine/src/Engine/ResourceManager.cpp

```diff
@@ -89,6 +89,26 @@ namespace Engine
 		}
 	}
 
+	//Save provided json object to a provided file. Defaults file name and path to "Data/data.json"
```

Comment:
Great explanation! Just adjust the 'provided file' to 'provided filename' to avoid confusion.

Reply:
good point, updated